examples/example_sim.py

```diff
@@ -4,11 +4,11 @@
 
 import sciris as sc
 import fpsim as fp
-import fp_analyses.senegal_parameters as sp
+import fpsim.locations.kenya.kenya as kp
 
 # Set options
 do_plot = True
-pars = sp.make_pars()
+pars = kp.make_pars()
 pars['n'] = 500 # Small population size
```

Comment:
@MObrien-IDM this is a bit temporary.  I'm using example_sim as a testing file as I make changes and need to access the kenya parameters to do it.   I ended up committing this change so I don't have to keep changing it each time.   I'd have to look at make_pars to see if the semantics work like that.

Reply:
@MObrien-IDM just looking at this briefly I don't think Kenya is automatically supported since it looks like valid locations are hard-coded right now?  @cliffckerr  this is question for you.
[
](https://github.com/fpsim/fpsim/blob/1eaa4e6d023956de1790f9789b4d2c97902b37f2/fpsim/parameters.py#L544)

Reply:
This is correct, it should be `fp.pars(location='kenya')`, there shouldn't be a need to import directly. And, yes, `parameters.py` should be updated with additional locations. We should also probably make it so the user can specify a path, e.g. `fp.pars(location='/my/custom/location/my_parameters.py'), although...at that point maybe the user should just call their script directly!

Reply:
Perfect!  This works.  parameters.py updated with kenya as an option